### ISSUE TITLE
Online demo

### ISSUE THREAD

**Issue by langston-barrett** [Ptival/PeaCoq#39]
Would be pretty slick :wink: 

I'm thinking something like https://try.jupyter.org/

**Comment by langston-barrett** [comment]
The blog post references a demo, but it doesn't work: http://goto.ucsd.edu:4242/

**Comment by Ptival** [comment]
Indeed, I haven't maintained the demo, and I don't have one at the moment.

Soon™ 🙃
